tests/Parser/Strings/StringContent.cpp

```diff
@@ -17,7 +17,7 @@ TEST(StringContent, dynamic_when_given_content_only_then_returns_false) {
 TEST(StringContent, dynamic_when_given_a_variable_then_returns_false) {
   StringContent content{};
 
-  content.add(0);
+  content.add(std::tuple<size_t, std::vector<std::string>>(0, {}));
```

Comment:
Why not use [`std::make_tuple`](https://en.cppreference.com/w/cpp/utility/tuple/make_tuple)? Same for all other occurrences.
```suggestion
  content.add(std::make_tuple(0, {}));
```